### ISSUE TITLE
@Url.Kentico().ImageUrl(...), not @Url.KenticoImageUrl(...)

### ISSUE THREAD

**Issue by jschrab-lc** [Kentico/training-kentico-k12#23]
### Brief bug description
I suspect a minor code error in `Views/Doctors/Detail.cshtml`. The following line...
```cshtml
<img src="@Url.KenticoImageUrl(Model.Data.Doctor.Photo.GetPath())" alt="Doctor's photo" />
```

### Repro steps

1. Add repo to project solution
2. Attempt to build
3. See error

### Expected behavior

I believe the correct code would be...
```cshtml
<img src="@Url.Kentico().ImageUrl(Model.Data.Doctor.Photo.GetPath(), SizeConstraint.Empty)" alt="Doctor's photo"/>
```
### Test environment

 - Version - Kentico v12.0

**Comment by Enngage** [comment]
Thank you for submitting the issue! In general you would be correct, but in this case it is intentional because we want to separate Kentico API from MVC project (and other libraries for that matter) as much as we can. For that reason we created an extension method using the `KenticoImageUrl` signature that you can find here: https://github.com/Kentico/training-kentico-k12/blob/master/MedioClinic/Extensions/UrlExtensions.cs 

Basically you wrap Kentico code in your own so that you have more control over it, you don't 'pollute' your views with Kentico code/namespace which then results in better maintainability/extensibility and upgradability.
